content/src/repository/extensions/DeploymentsRepository.ts

```diff
@@ -232,6 +232,19 @@ export class DeploymentsRepository {
       return transaction.batch(updates)
     })
   }
+
+  async getDeploymentByHash(contentHash: string): Promise<{ entityId: string }> {
```

Comment:
I'd use a more specific name, something like `getDeploymentsFromContentHash`.

Reply:
Yes, this is pre refactor, will rename, thanks!